bin/prettier.js

```diff
@@ -69,8 +68,8 @@ const filepatterns = argv["_"];
 const write = argv["write"];
 const stdin = argv["stdin"] || (!filepatterns.length && !process.stdin.isTTY);
 const ignoreNodeModules = argv["with-node-modules"] === false;
+const ignoreNodeModulesGlobs = ["**/node_modules/**", "./node_modules/**"];
```

Comment:
@k15a no, it doesn't work with monolytics repo (`lerna`), some repositories use this structure